web/docs/setup/configure.md

```diff
@@ -174,9 +174,11 @@ Static plugins do not require installation since they are compiled into VAST.
 ### Load plugins
 
 The onfiguration key `vast.plugins` specifies the list of plugins that should
-load at startup. The `bundled` and `all` plugin names are reserved. When
-`bundled` is specified VAST loads all plugins built along with it. When `all` is
-specified VAST loads all available plugins in the configured plugin directories.
+load at startup. The `all` plugin name is reserved. When `all` is specified
+VAST loads all available plugins in the configured plugin directories. If no
+`vast.plugins` key is specified, VAST will load `all` plugins by default.
+To load no plugins at all, specify a `vast.plugins` key with no
+values in the configuration file or use the launch parameter `--plugins=[]`.
```

Comment:
When testing this, please make sure it works for all these scenarios (and combinations of them)
1. `vast --plugins= start`
2. `vast --plugins=[] start`
3. `VAST_PLUGINS= vast start`
4. `VAST_PLUGINS=[] vast start`
5. `vast start` with `vast: { plugins: [] }` in your config file.

The empty value should be supported as well, since the documentation states that we take a comma-separated list of plugins.

Reply:
1 ends up in a `parser_error`
3 and 4 end up in `all` being included despite an empty value being specified.

Yep, this is probably the specified "fix merging of configuration options with a known list type to allow for distinguishing between not specified and empty list (`plugins = []`)" task I'm working on.

Reply:
As of now, my current implementation has minimal `configuration.cpp` changes. @patszt feel free to check if conflicts would arise (in the future).